SoftLayer/managers/ordering.py

```diff
@@ -509,7 +509,7 @@ def place_quote(self, package_keyname, location, item_keynames, complex_type=Non
         return self.order_svc.placeQuote(order)
 
     def generate_order(self, package_keyname, location, item_keynames, complex_type=None,
-                       hourly=True, preset_keyname=None, extras=None, quantity=1):
+                       hourly=True, preset_keyname=None, extras=None, quantity=None):
```

Comment:
I'd say leave this as `quantity=1` that way you can just leave `order['quantity'] = quantity` and can avoid the extra if statement

Reply:
Done